### DISCUSSION TITLE
Display error code for traditional Chinese in .ipynb file viewer

### DISCUSSION THREAD

**Discussion by roberthsu2003** [community/community#37292] [Repositories]
https://github.com/roberthsu2003/PythonForDataAnalysis/blob/master/pandas的基礎功能/README.ipynb

<img width="1192" alt="截圖 2022-10-26 下午12 46 10" src="https://user-images.githubusercontent.com/1614274/197936880-d385f17c-ba50-4518-a1b1-33b507329c81.png">

**Comment by donghui-0126** [comment]
i use Korean. i have same problem. you still have same problem???
i search solution of this problem, but i have still problem. 
 i think error of github or jupyter notebook.

**Reply by roberthsu2003** [reply]
it is fine right now
